applications/CoSimulationApplication/python_scripts/solver_wrappers/sdof/sdof_solver.py

```diff
@@ -0,0 +1,278 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+
+# CoSimulation imports
+from KratosMultiphysics.CoSimulationApplication.function_callback_utility import GenericCallFunction
+
+# Other imports
+import numpy as np
```

Comment:
It would be nice to check if there is numpy or not (also you can add that this needs numpy in the class description) and throw an error ?